### PR TITLE
Holonomic 6-DoF Free-Flying Space Robot (Astrobee)

### PR DESCRIPTION
**Pull request by abhikamath**
Example
-------

   - [x] There are no merge conflicts.
   - [x] Examples rst file is placed in `docs/examples/`.
   - [x] An entry with a 200x200 pixel image has been added to `docs/index.rst`.
   - [x] RsT and image filenames are kebab-case (lower case hyphen-separated).
   - [x] jupyter_sphinx RsT directives are used to ensure example is executed on build.
   - [x] jupyter_sphinx `ipynb` and `py` download links are included.
   - [x] The example has adequate text and figures (preferably SVG) explaining
     the problem.
   - [x] All reviewer comments have been addressed.

### CODE REVIEW

File: docs/examples/Space_Robot.rst [hunk 1]
```diff
@@ -0,0 +1,792 @@
...
+3D Visualization
+----------------
+
+.. code:: ipython3
+
+    from pydy.viz.shapes import Cube, Cylinder, Sphere, Plane
+    from pydy.viz.visualization_frame import VisualizationFrame
+    from pydy.viz import Scene
+    from ipywidgets import Image, Video
+    import pythreejs as pjs
```
Comment (moorepants) [inline docs/examples/Space_Robot.rst:463]: The hack to use pythreejs is no longer needed. Please update to use the simpler form in the development version of pydy. The other documentation examples show how to do this:

https://pydy.readthedocs.io/en/latest/index.html#examples

**Comment by moorepants** [comment]
Remove the mov file and DS_store file.

File: docs/examples/Space_Robot.rst [hunk 2]
```diff
@@ -0,0 +1,538 @@
+=============================================
+Astrobee: A Holonomic Free-Flying Space Robot
+=============================================
```
Comment (moorepants) [inline docs/examples/Space_Robot.rst:3]: @abhikamath Both of these examples could use, at minimum, a paragraph introduction explaining what the model is, what it does, and any basic intro information.

File: docs/examples/Space_Robot.rst [hunk 3]
```diff
@@ -0,0 +1,538 @@
...
+    states = sys.integrate()
+
+.. jupyter-execute::
+
+    import matplotlib as mpl
+    mpl.rcParams['figure.dpi'] = 200
+    mpl.rc('font',**{'family':'serif','sans-serif':['Computer Modern Roman']})
+    ## for Palatino and other serif fonts use:
+    #rc('font',**{'family':'serif','serif':['Palatino']})
+    mpl.rc('text', usetex=True)
```
Comment (moorepants) [inline docs/examples/Space_Robot.rst:307]: I think this is causing issues. We don't have latex installed on the CI system, so removing all these mpl.rc settings is probably preferable.

**Comment by moorepants** [comment]
@abhikamath I've made a few additions. See my changes, as you'll want to do these for the other example too: wrap text to 70 characters, add a 200px x 200px image (or gif) to the index.rst, merge in master so the Box works.

**Comment by moorepants** [comment]
This one is ready to go in! I'll let the tests complete and then merge. Nice work!

**Comment by moorepants** [comment]
Thank Abhi! Merged!

**Comment by moorepants** [comment]
Here it is:

https://pydy.readthedocs.io/en/latest/examples/astrobee.html

**Comment by VirtualAcoustics** [comment]
@abhikamath Nice example!

**Pull request merged**